source/includes/stackpath/_sites.md

```diff
@@ -442,6 +442,7 @@ curl -X POST \
 
 ```json
 {
+  "purgeType":"PATH",
```

Comment:
I don't think the purge type should be needed. The API supports other types than URL and PATH, and you can have a mix of both.

Reply:
Changed it to optional field. Default type is URL now
74776cc